src/akkudoktoreos/devices/battery.py

```diff
@@ -6,73 +6,72 @@
 from akkudoktoreos.utils.utils import NumpyEncoder
 
 
-def max_ladeleistung_w_field(default: Optional[float] = None) -> Optional[float]:
+def max_charge_power_field(default: Optional[float] = None) -> Optional[float]:
+    """Creates a field for maximum charging power."""
     return Field(
         default=default,
         gt=0,
-        description="An integer representing the charging power of the battery in watts.",
+        description="A float representing the maximum charging power of the battery in watts.",
     )
 
 
-def start_soc_prozent_field(description: str) -> int:
+def initial_soc_percent_field(description: str) -> int:
+    """Creates a field for the initial state of charge in percentage."""
     return Field(default=0, ge=0, le=100, description=description)
 
 
-class BaseAkkuParameters(BaseModel):
-    kapazitaet_wh: int = Field(
-        gt=0, description="An integer representing the capacity of the battery in watt-hours."
-    )
-    lade_effizienz: float = Field(
-        default=0.88,
-        gt=0,
-        le=1,
-        description="A float representing the charging efficiency of the battery.",
+class BatteryParameters(BaseModel):
+    """Base model for battery parameters."""
+
+    capacity_wh: int = Field(gt=0, description="The capacity of the battery in watt-hours.")
+    charging_efficiency: float = Field(
+        default=0.88, gt=0, le=1, description="Charging efficiency of the battery."
     )
-    entlade_effizienz: float = Field(default=0.88, gt=0, le=1)
-    max_ladeleistung_w: Optional[float] = max_ladeleistung_w_field()
-    start_soc_prozent: int = start_soc_prozent_field(
-        "An integer representing the state of charge of the battery at the **start** of the current hour (not the current state)."
+    discharging_efficiency: float = Field(default=0.88, gt=0, le=1)
+    max_charge_power_w: Optional[float] = max_charge_power_field()
+    initial_soc_percent: int = initial_soc_percent_field(
+        "The initial state of charge of the battery in percentage."
     )
-    min_soc_prozent: int = Field(
+    min_soc_percent: int = Field(
         default=0,
         ge=0,
         le=100,
-        description="An integer representing the minimum state of charge (SOC) of the battery in percentage.",
+        description="The minimum state of charge (SOC) of the battery in percentage.",
     )
-    max_soc_prozent: int = Field(default=100, ge=0, le=100)
+    max_soc_percent: int = Field(default=100, ge=0, le=100)
+
 
+class PVBatteryParameters(BatteryParameters):
+    """Model for battery parameters specific to PV systems."""
 
-class PVAkkuParameters(BaseAkkuParameters):
-    max_ladeleistung_w: Optional[float] = max_ladeleistung_w_field(5000)
+    max_charge_power_w: Optional[float] = max_charge_power_field(5000)  # why?
```

Comment:
can remove the default, was taken from the initial code where this was hardcoded